backend/src/api/events.js

```diff
@@ -0,0 +1,196 @@
+const express = require('express')
+const mongodb = require('mongodb')
+const router = express.Router()
+const { errorWrap } = require('../middleware')
+const { Event } = require('../models')
+
+// get all events
+router.get(
+  '/',
+  errorWrap(async (req, res) => {
+    let events = await Event.find()
+    res.json({
+      code: 200,
+      message: '',
+      result: events,
+      success: true
```

Comment:
Add trailing comma to all multiline statements in Javascript. Just for reference:

1. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas
2. https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
agreed ^

Reply:
so yarn format gets rid of trailing commas rip we'd have to change that

Reply:
true, I forgot thats why we don't have them anywhere